client/blocks/color-scheme-picker/docs/example.jsx

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import ColorSchemePicker from 'blocks/color-scheme-picker';
+
+class ColorSchemePickerExample extends PureComponent {
+	static displayName = 'ColorSchemePicker';
+
+	state = {
+		selectedColorScheme: null,
+	};
+
+	handleColorSchemeSelection = colorScheme => {
+		this.setState( { selectedColorScheme: colorScheme } );
+	};
+
+	render() {
+		return (
+			<Card>
+				<ColorSchemePicker
+					temporarySelection={ true }
+					onSelection={ this.handleColorSchemeSelection }
+				/>
+
+				{ this.state.selectedColorScheme &&
+					<Notice
+						text={ `Selected color scheme: ${ this.state.selectedColorScheme }` }
+						showDismiss={ false }
+					/> }
```

Comment:
It was on its own line initially but got moved up by Calypso prettier. If that is still a requirement we'd either have to not run prettier on the file or the prettier config would need to support that. Given that it's a readability nice to have I assume it's OK to leave it this way?